AUDIT_ADVERSARIAL_BRANCH.md

```
@@ -0,0 +1,468 @@
+- `/home/lrleon/Aleph-w/Examples/branch_and_bound_assignment_example.cc`
+- `/home/lrleon/Aleph-w/Examples/branch_and_bound_artificial_example.cc`
+- `/home/lrleon/Aleph-w/Examples/negamax_tictactoe_example.cc`
+- `/home/lrleon/Aleph-w/Examples/alpha_beta_connect3_example.cc`
+- `/home/lrleon/Aleph-w/Examples/adversarial_artificial_example.cc`
+- `/home/lrleon/Aleph-w/Examples/ida_star_grid_example.cc`
+- `/home/lrleon/Aleph-w/Examples/state_search_framework_example.cc`
+- `/home/lrleon/Aleph-w/Examples/state_search_benchmark.cc`
+
+**Documentación** (5):
+- `/home/lrleon/Aleph-w/docs/state_search_framework.md`
+- `/home/lrleon/Aleph-w/docs/branch_and_bound_framework.md`
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)